pkg/config/config.go

```diff
@@ -59,6 +43,10 @@ func NewFromFlags() (*Config, error) {
 
 	flag.Parse()
 
+	if _, err := printer.ParsePrinter(config.Output); err != nil {
+		return nil, fmt.Errorf("failed to validate argument output")
```

Comment:
I suggest not hiding the error, wrap it instead as below:
```suggestion
		return nil, fmt.Errorf("failed to validate argument output: %w", err)
```